docs/reference/aggregations/pipeline.asciidoc

```diff
@@ -133,6 +144,7 @@ used here to filter out buckets which contain no buckets for an inner terms aggr
 
 [source,js]
 --------------------------------------------------
+POST /sales/_search
```

Comment:
I'm ok either way. I figured POST was a more compliant thing from reading [this](http://stackoverflow.com/questions/978061/http-get-with-request-body). I think we have a similar issue but I can't find it at the moment.